src/core/a-scene.js

```diff
@@ -93,24 +89,12 @@ var AScene = module.exports = registerElement('a-scene', {
         var resizeCanvas = this.resizeCanvas.bind(this);
         this.setupKeyboardShortcuts();
         this.attachFullscreenListeners();
```

Comment:
Fullscreen events should be ignored when vr-mode is forced to VR?    Stereo rendering resets to mono when exiting fullscreen.